docs/new-feature-tutorial.md

```diff
@@ -289,3 +296,23 @@ Any code needed to update the UI should be placed in
 function. This ensures the appropriate code will run even if the
 changes are made in another browser window. In this example most of
 the changes are on the backend, so no UI updates are required.
+
+### Update documentation
+
+Now it is time to document your feature. This feature adds new
```

Comment:
"Now it is time" is kind of vague, maybe you should replace it with "After you add a new view, you should document your feature"?